.travis.yml

```diff
@@ -31,7 +31,12 @@ before_install:
 # TODO: https://github.com/NixOS/nixpkgs/issues/21200
 - travis_retry curl -L https://www.stackage.org/stack/$TRAVIS_OS_NAME-x86_64 | tar
   xz --strip-components=1 -C ~/.local/bin
+
 script:
+- pushd ~/.local    # any path that doesn't have a stack.yaml
```

Comment:
No, because we override the `haskell-src-exts` version in the project and hlint doesn't work with the older version we're using. I'll add a comment to `.travis.yml` about that.